Dockerfile.composer

```diff
@@ -0,0 +1,16 @@
+FROM composer:1.6.3
+
+ARG composer_dev
+
+COPY composer.json \
+    composer.lock \
+    ./
+
+RUN echo "composer --no-interaction install ${composer_dev} --ignore-platform-reqs --no-autoloader --no-scripts --no-suggest"
+RUN composer --no-interaction install ${composer_dev} --ignore-platform-reqs --no-autoloader --no-scripts --no-suggest
+
+COPY tests/ tests/
```

Comment:
is there a reason for the order of `COPY` statements?

Reply:
Hypothesis that `src` is more likely to change. But 🤷‍♂️.